sphinx/conf.py

```diff
@@ -58,8 +58,8 @@
 
 html_title = "skore"
 
-html_theme_options = {
-    "announcement": (
-        "https://raw.githubusercontent.com/soda-inria/hazardous/main/doc/announcement.html"
-    ),
-}
+# html_theme_options = {
```

Comment:
do we need to keep this comment? If it's just because we might re-use this code later, git is here to keep history, let's not overload the code.

Reply:
sure, deleted